SSClient.py

```diff
@@ -135,20 +136,34 @@ def get_secrets_list(username: str, password: str, otp: int):
 def _get_secrets_list(username: str, password: str, otp: int):
     """Lists basic information in of secrets that can be used to
        fetch the actual secret"""
+    
+    # setup token
     token = _get_token(username, password, otp)
     try:
         session = get_legacy_session()
         headers = {"Authorization": f"Bearer {token}"}
-        response = session.get(f"{OHSU_SECRET_SERVER_ENDPOINT}/api/v2/secrets",
-                               headers=headers)
+        more_records = True
+        start = 0
+
+
+        print(f"id{'':13}name{'':36}secretTemplateName{'':2}folderId{'':2}lastUpdated")
+        while more_records:
+            response = session.get(f"{OHSU_SECRET_SERVER_ENDPOINT}/api/v2/secrets",
```

Comment:
Let's double check if OK to include "Secret Server" here or if we should hide it. I think it's OK but not 100% sure.

Reply:
Discussed offline that it was alright